get_defs.py

```diff
@@ -5,40 +5,45 @@ def api_call(word: str) -> list:
     
     trial = 0
     time.sleep(0.5)  #0.4 sleep with 2 attempts has had best results
+
     while trial < 2:
-        trial +=1
+
         try:
             url = f"https://api.dictionaryapi.dev/api/v2/entries/en/{word}"
             response = requests.get(url)
 
             if response.status_code == 200:
                     data = response.json()
-                    #print(f"Data found for {word}")
-                    return data
-            
+                    return data          
             else:
-                print(f"This was the error code: {response.status_code}")
-        except:
+                trial +=1
+                raise Exception(f"non-200 status code returned for {word}: {response.status_code}")
+            
+
+        except Exception:
             print(f"Bad Request - no relvant url found for {word}")
             return None
```

Comment:
2 things;

1. this message lies!
2. the return of None is probably problematic as to higher level control flow.

Reply:
I'm currently trying (bool) False instead. Possible issue being 0 == False so let me know if there's a standard/norm to follow